core/framework/config.py

```
@@ -0,0 +1,168 @@
+        Load configuration with hierarchy.
+        
+        Args:
+            processor_name: Name of the processor
+            override: Runtime configuration overrides
+            
+        Returns:
+            Loaded configuration object
+        """
+        # Start with base config
+        config = ConfigManager._load_base()
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)